wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -531,6 +553,16 @@ impl PhysicalDeviceFeatures {
             );
         }
 
+        if let Some((ref f16_i8, ref bit16)) = self.shader_float16 {
+            features.set(
+                F::SHADER_FLOAT16,
+                f16_i8.shader_float16 != 0
+                    && f16_i8.shader_int8 != 0
```

Comment:
similarly, why are we checking for int8?

Reply:
Oh, my mistake.